database/seeders/postgresSeeder.js

```diff
@@ -0,0 +1,29 @@
+// postgres seeding script
+const { Rooms, Photos } = require('../postgres/models/index.js');
+const { csvGenerator } = require('./csvDataGenerator.js');
+
+const postgresSeeder = async (recordsToCreate) => {
+  const startTime = new Date().getTime();
+
+  try {
+    await Rooms.sync({ force: true });
+    await Photos.sync({ force: true });
+    console.log('The table for the Rooms and Photos model was just (re)created!');
+
+    // batch number variable to create and save (change as desired for performance testing)
+    const recordsPerBatch = 100000;
```

Comment:
How did you come up with this number?

Reply:
Similar to my other script, I tested out different batch sizes. Prior to my postgres schema refactor, this 100k batch size generated 10M records < 30 minutes. 

However, after refactoring my schema to have two tables to add complexity, the generation time almost doubled to roughly 1 hour. Since this was in the ballpark of 1 hour, I didn't attempt to change. Maybe on deployment I might try a lower batch size, like I did with the couchDB script.